src/analysis/retail/warlock/destruction/modules/soulshards/SoulShardDetails.tsx

```diff
@@ -9,7 +10,6 @@ import BoringSpellValueText from 'parser/ui/BoringSpellValueText';
 import Statistic from 'parser/ui/Statistic';
 import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
 
-import 'analysis/retail/warlock/SoulShardDetails.css';
```

Comment:
are you certain that this css import is unused? I don't see the css file itself deleted.

Reply:
I don't think  this file currently exists? I don't see it in the directory, and I get this error when I don't remove it
**Module not found: Error: Can't resolve 'analysis/retail/warlock/SoulShardDetails.css'**

Reply:
The other warlock PR removed it I think. Disregard then